yt_dlp/extractor/yandexvideo.py

```diff
@@ -147,7 +147,7 @@ def _real_extract(self, url):
 
 
 class YandexVideoPreviewIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?yandex\.ru/video/preview(?:/?\?.*?filmId=|/)(?P<id>\d+)'
+    _VALID_URL = r'(?x)https?://(?:www\.)?yandex\.\w+(?:\.\w+)?/video/preview(?:/?\?.*?filmId=|/)(?P<id>\d+)'
```

Comment:
The first part of top level domain is fine as `\w+`. But we should hardcode the second `\w+`. Currently this matches `yandex.somesite.com` etc which we don't want

Reply:
Also, `(?x)` is unnecessary

Reply:
I use `\w{2,3}` for the first and hardcoded the second, just to be sure.